app/helpers/arcgis_helper.rb

```diff
@@ -0,0 +1,12 @@
+module ArcgisHelper
+  ##
+  # Generates an ArcGIS.com viewer url with params that can open content
+  # directly
+  # See: https://doc.arcgis.com/en/arcgis-online/reference/use-url-parameters.htm
+  def arcgis_link(arcgis_url)
```

Comment:
For consistency, it would probably be good pluralize the param name here: `arcgis_urls`.